inventory.py

```diff
@@ -167,14 +180,115 @@ def check_if_valid_user_id(id: int, user_ids: Sequence[int]) -> bool:
         return False
 
 
+def get_next_page_url(response):
+    """
+    get the next page url from the Http response headers
+    :param response:
+    :type response: requests
+    :return: next_page_url
+    :rtype: str
+    """
+    logging.debug(get_next_page_url.__name__ + '() called')
+    results = response.result().links
+    if not results:
+        logging.debug('The api call do not have Link headers')
+        return None
+
+    for page in results:
+        if 'next' in page:
+            url_ = results['next']['url']
+            published_course_next_page_list.append(url_)
+            logger.info(f"Pagination size {len(published_course_next_page_list)} for published_at date")
+            break
```

Comment:
Can you just check for "next" in `result.keys()` rather than looping here?

Reply:
Done